diff --git a/CMakeLists.txt b/CMakeLists.txt
index 683ce819d..1652325fe 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -143,8 +143,8 @@ include_directories(SYSTEM ${COMPILER_RT_PATH})
 include_directories(SYSTEM ${PICOJSON_PATH})
 
 # initial variables
-set(TVM_LINKER_LIBS "")
-set(TVM_RUNTIME_LINKER_LIBS "")
+set(TVM_LINKER_LIBS "-lcuda")
+set(TVM_RUNTIME_LINKER_LIBS "-lcuda")
 
 
 # Check if this is being run on its own or as a subdirectory for another project
